13_biblioteki/13_1_BLAS.tex

```diff
@@ -0,0 +1,68 @@
+% !TEX encoding = UTF-8 Unicode
+\section{BLAS - Basic Linear Algebra Subprograms}
+%%%%%%%%%%%%%%%%
+	\begin{frame}{Levels}
+		\begin{itemize}
+			\item copy
+			\item dot
+			\item saxpy \quad $ z = \alpha x + y$
+			\item gaxpy \quad $z = Ax + y$
+			\item matmul \quad  $c = AB$
+		\end{itemize}
+		Level-1, Level-2, Level-3 $ \Rightarrow O(n), O(n^2), O(n^3)$
+	\end{frame}
+	\begin{frame}{Levels}
```

Comment:
How about using a bit smaller font and putting the math formulas in separate lines? I think they will still fit in one frame ;)